scenario_player/environment/arbitrum.json

```
@@ -0,0 +1,15 @@
+{
+    "environment_type": "development",
+    "pfs_with_fee": "https://pfs.arbitrum.env.raiden.network",
+    "eth_rpc_endpoints": [
+        "https://rinkeby.arbitrum.io/rpc"
+    ],
+    "transfer_token": "0x99C1aE38292791a52E26eE6AFdAeC69563F9b5C3",
+    "pfs_fee": 100,
+    "ms_reward_with_margin": 88000000000000000000,
+    "settlement_timeout_min": 40,
+    "raiden_client": "/Users/fernando/projects/light-client/raiden-cli/raiden",
+    "wait_short": 30,
+    "wait_long": 60,
+    "gas_price": "rpc"
```

You'll probably need:
```suggestion
    "gas_price": "rpc",
    "matrix_servers": [ "auto", "auto", "auto", "auto" ]
```
`auto` let nodes detect its servers (which should work with LC); 4 of them will make it work with some scenarios which expect at least 4 servers to be specified